.github/workflows/static.yaml

```diff
@@ -24,8 +24,21 @@ jobs:
     # Therefore, this step should remain the first one that is executed after
     # the Go setup, in case other steps are added to this job in the future.
     - name: Lint Go code
+      id: lint
       uses: golangci/golangci-lint-action@v3
 
+    # (hack) By default, errors are reported to the "Files changed" tab on PRs.
+    # In case errors were found, run the linter a second time but don't direct
+    # the output to GitHub, so that they are also visible in the execution logs
+    # of the workflow, with line numbers.
+    # Ref. https://github.com/golangci/golangci-lint-action/issues/119#issuecomment-981090648
+    - name: Lint Go code (debug)
+      if: ${{ failure() && steps.lint.outcome == 'failure' }}
+      uses: golangci/golangci-lint-action@v3
+      with:
+        args: "--out-${NO_FUTURE}format colored-line-number"
```

Comment:
I just made a [funny discovery](https://github.com/golangci/golangci-lint-action/issues/119#issuecomment-1096917139).
This workflow steps actually executes the following command:

```
golangci-lint run --out-format=github-actions --out-format=colored-line-number
```

Turns out this effectively logs errors to both GitHub _and_ the workflow logs 😅 
So we might not even need this second `if: faillure()` step.